wasmcloud_host/chart/Chart.yaml

```diff
@@ -15,6 +15,6 @@ icon: https://github.com/wasmCloud/wasmcloud.com-dev/raw/main/static/images/wasm
 
 type: application
 
-version: 0.6.1
+version: 0.6.2
```

Comment:
So there is a weird bug with the chart publisher stuff that it won't detect changes from the previous version if run when a tag is cut. Can you please manually trigger a chart build before tagging?

Reply:
Will do!